packages/priority-overflow/src/overflow-manager.ts

```diff
@@ -0,0 +1,226 @@
+import debounce from './debounce';
```

Comment:
no reason really to unit test this, since the actual behaviour is so dependent on the browser.

Browser tests will be done in `@fluentui/react-priority-overflow`